README.md

```diff
@@ -378,6 +380,42 @@ ValueError: Specify tags before launching a multirun!
 
 <br>
 
+## Project Guidelines
```

Comment:
Very cool! I find it easy to understand, well done. Should we add something about the task and how that links models and dataset?

Reply:
I think what I should do is add documentation about the entire config code and how it interlinks. I'll do that in the future :smile_cat: